src/components/CheckboxGroup/checkbox.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Checkbox(props) {
+    const { id, label, disabled, isSelected, onChange } = props;
+    return (
+        <div className="rainbow-checkbox" role="presentation" onClick={() => onChange(id, !isSelected, disabled)}>
+            <input
+                id={id}
+                type="checkbox"
+                value={id}
+                checked={isSelected}
+                disabled={disabled}
+                readOnly />
+            <label className="rainbow-checkbox__label" htmlFor={id}>
+                <span className="rainbow-checkbox_faux" />
+                <span className="rainbow-form-element__label">{label}</span>
```

Comment:
rainbow-form-element__label class does not follow the format:
rainbow-component-name-modifiers